src/features/content-answers/ContentAnswers.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+
+import ContentAnswersOpenButton from './ContentAnswersOpenButton';
+
+const ContentAnswers = () => {
+    const [openContentAnswersModal, setOpenContentAnswersModal] = useState(false);
+
+    const handleClick = () => {
+        setOpenContentAnswersModal(!openContentAnswersModal);
```

Comment:
you should use the prevState in this calculation:
```suggestion
        setOpenContentAnswersModal(prevState => !prevState);
```